server/src/main/java/io/spine/server/procman/LifecycleRules.java

```diff
@@ -63,8 +63,12 @@ public LifecycleRules() {
     /**
      * Configures the repository to archive its entities when certain events/rejections occur.
      *
-     * <p>Subsequent calls to this method do not clear the previously added event classes (i.e. not
-     * override each other).
+     * <p>Subsequent calls to this method do not clear the previously added event classes,
+     * i.e, in order not to override each other. For example, if we make two subsequent calls:
+     * `lifecycle.archiveOn(Event1.class)`;
+     * `lifecycle.archiveOn(Event2.class)`;
+     * The repository will archive when either `Event1` or `Event2` occurs and  and not only when `Event2` occurs.
```

Comment:
We probably can use `{@code Event1}` and `{@code Event2}` here either.

Also, this line seems to be longer than allowed 100 symbols.